src/SourceGenerators/Uno.UI.Tasks/Content/uno.ui.tasks.assets.targets

```diff
@@ -171,6 +171,11 @@
 			BeforeTargets="GetCopyToOutputDirectoryItems">
 
 		<ItemGroup>
+			<!-- Explicit exclusion of the assets when building for a ASP.NET hosted server project -->
+			<_HasUnoBootstrapServerReference Include="@(PackageReference)" Condition="'%(Identity)'=='Uno.Wasm.Bootstrap.Server'" />
```

Comment:
nit: The "Has" in the same is quite confusing. I think naming it `_UnoBootstrapServerReference` would make it less confusing.

Reply:
Agreed!